docs/widgets/firewall/index.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Firewall Monitoring'
+description: 'Displays a summary of firewalls.'
+hide_title: true
+---
+
+import { WidgetHeader } from '@site/src/components/widgets/header';
+import { AddingWidget } from '@site/src/components/widgets/adding';
+import { WidgetIntegrations } from '@site/src/components/widgets/integrations';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { firewallWidget } from '.';
+import { opnsenseIntegration } from '../../integrations/opnsense';
+
+
+<WidgetHeader
+  widget={firewallWidget}
+  categories={['Firewall']}
+/>
+
+### Screenshots
+
+:::note
+Screenshots will follow soon.
```

Comment:
I assume you don't have it running and can't take any?

Reply:
Yes exactly and the original creator said he is going to make docs (but we need it for the docs integration list